### DISCUSSION TITLE
How to implement logo marks for different models on promotional pages?

### DISCUSSION THREAD

**Discussion by flyfox666** [open-webui/open-webui#2851] [Q&A]
I see that selecting different models in the showcase page can have different model logos as follows: 

![image](https://github.com/open-webui/open-webui/assets/121539277/7e066a13-4cad-40f1-9092-bbb2fec291a1)

Would like to know how I can set this up, I'm using groq

**Comment by justinh-rahb** [comment]
![Screenshot 2024-06-05 at 12 21 51 PM](https://github.com/open-webui/open-webui/assets/52832301/00954e9b-d2f7-49be-8d58-435000388f2d)

#### Here's the exports for my Groq models:
[gemma-7b-it-1717604460772.json](https://github.com/user-attachments/files/15593555/gemma-7b-it-1717604460772.json)
[llama3-70b-8192-1717604463829.json](https://github.com/user-attachments/files/15593561/llama3-70b-8192-1717604463829.json)
[llama3-8b-8192-1717604466596.json](https://github.com/user-attachments/files/15593563/llama3-8b-8192-1717604466596.json)
[mixtral-8x7b-32768-1717604469997.json](https://github.com/user-attachments/files/15593565/mixtral-8x7b-32768-1717604469997.json)

#### Import them from **Workspace > Models**:
![Screenshot 2024-06-05 at 12 23 02 PM](https://github.com/open-webui/open-webui/assets/52832301/256aa0fe-e9d5-474f-8839-d0e36b13ab04)

#### You can edit model names, descriptions, tags, and images to your preference:
![Screenshot 2024-06-05 at 12 24 45 PM](https://github.com/open-webui/open-webui/assets/52832301/479df3ec-61f7-4217-a0cd-2694fc3dc16c)

**Reply by flyfox666** [reply]
Oh，really thanks

**Comment by naudor** [comment]
Can you tell us the specification of the images, please? I can't center new logo, don't matter the size.
Thanks

**Comment by rfernandez760** [comment]
Take a look at this site. https://custom.typingmind.com/tools/model-icons

**Comment by nov3o** [comment]
Is there a way to make an automatic icon pulling upon model addition via some hook?
